src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -577,9 +577,17 @@ public override void Execute(ICpu cpu)
             object popValue = cpu.PopValue();
 
             var specialValue = popValue as ISuffixed;
+            
             if (specialValue == null)
             {
-                throw new Exception(string.Format("Values of type {0} cannot have suffixes", popValue.GetType()));
+                if (popValue is String)
+                {
+                    specialValue = new StringValue((String)popValue);
```

Comment:
This works as a temporary for now, and is pretty clever now that I understand what you're doing (this is invoked every time you attempt to use the colon operator).  I think as a future refactor it should be changed to actually being on the stack as a StringValue, for other reasons, but doing so takes a bit of work (i.e. it has to be stored in KSM files that way, it has to be wrapped everywhere in the code we're using it. etc).  The reason for my wanting it is to help make kOS able to do types properly (i.e. have an isType(foo) operator), where all types in the system are made by us.  But that's a larger project, and in order to just get string operations working at all, I think your solution is a good 'shim' fix for now.